pom.xml

```diff
@@ -628,6 +628,7 @@
                         <exclude>Jenkinsfile</exclude>
                         <exclude>**/codestyle/*</exclude>
                         <exclude>**/resources/META-INF/**</exclude>
+                        <exclude>**/test/**</exclude>
```

Comment:
`test` directory is general,  maybe misjudge

Reply:
> `test` directory is general, maybe misjudge

OK, I have renamed the directory's name from test to .tmp